src/api/app/policies/project_policy.rb

```diff
@@ -39,6 +39,10 @@ def source_access?
     User.admin_session? || !record.disabled_for?('sourceaccess', nil, nil)
   end
 
+  def create_new?
+    record.new_record? && create?
```

Comment:
I believe this should go in `create?`. What is the reason for having `create_new?`?

Reply:
hmm, fair question. create is create not update. But other controllers use Project.new, which always makes this true, so this needs to be looked at as well.